android/app-newm/src/androidTest/AndroidManifest.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest
+    package="io.newm.test"
```

Comment:
Interesting, I had removed the `package` name in favor of the namespace in gradle dsl.